codice/alliance
# [1.1.x] CAL-464 Removing error page mappings from Alliance

#### What does this PR do?
Remove unnecessary error page mappings that are done with the DDF platform-error-page-injector bundle done in [DDF-4033](https://codice.atlassian.net/browse/DDF-4033)

#### Who is reviewing it? 
@bakejeyner @brjeter @blen-desta 
(please choose AT LEAST two reviewers that need to approve the PR before it can get merged; if a component team is listed, at least one of its members needs to approve)
#### Choose 2 committers to review/merge the PR.
(please choose ONLY two committers from below, delete the rest)

@clockard
@coyotesqrl

#### How should this be tested?
1) Install this [DDF:4033](https://github.com/codice/ddf/pull/3621)
2) Build and start Alliance
3) Should see a successful build and no errors related to bundle wiring or error:mappings in the logs

#### Any background context you want to provide?
#### What are the relevant tickets?
[DDF-4033](https://codice.atlassian.net/browse/DDF-4033)
[CAL-464](https://codice.atlassian.net/browse/CAL-464)

#### Screenshots (if appropriate)
#### Checklist:
